scalatest/src/main/scala/org/scalatest/tools/Framework.scala

```diff
@@ -737,12 +739,14 @@ class Framework extends SbtFramework {
         paths.exists(path => td.fullyQualifiedName.startsWith(path) && td.fullyQualifiedName.substring(path.length).lastIndexOf('.') <= 0)
       }
       
-    def tasks(taskDefs: Array[TaskDef]): Array[Task] = 
-      for { 
+    def tasks(_taskDefs: Array[TaskDef | Null] | Null): Array[Task | Null] = {
+      val taskDefs = _taskDefs.nn.map(_.nn)
```

Comment:
**Overridden java array contents are never null**
Overridden array field of type `Array[T | Null]` should have type `Array[T]`. Use `.map(_.nn)` to fix.